src/VisualStudio/Core/Impl/CodeModel/ProjectCodeModelFactory.cs

```diff
@@ -41,14 +41,13 @@ internal sealed class ProjectCodeModelFactory : IProjectCodeModelFactory
             VisualStudioWorkspace visualStudioWorkspace,
             [Import(typeof(SVsServiceProvider))] IServiceProvider serviceProvider,
             IThreadingContext threadingContext,
-            IForegroundNotificationService notificationService,
             IAsynchronousOperationListenerProvider listenerProvider)
+            : base(threadingContext)
```

Comment:
Can we add an `assertIsForeground: false`. just to be explicit this isn't asserting the constructor is created there? I can never remember what the default is.

Reply:
the default is `:false`.

Reply:
added.!